### PR TITLE
docs(releases): add release notes for v0.31 to documentation

### PR DESCRIPTION
**Pull request by hassankhan**

### CODE REVIEW

File: docs/content/Release-Notes/0-31.mdx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+We’ve been busy since the last release adding support for downstream tools. Cube
+now connects to:
+
+- [Appsmith](https://cube.dev/docs/config/downstream/appsmith)
+- [Tableau](https://cube.dev/docs/config/downstream/tableau/)
+- [Microsoft PowerBI](https://cube.dev/docs/config/downstream/powerbi/)
+- Thoughtspot
+- Amazon QuickSight
+- Sigma Computing
```
Comment (hassankhan) [inline docs/content/Release-Notes/0-31.mdx:87]: @keydunov We don't currently have pages in our documentation for these

Comment (keydunov) [reply docs/content/Release-Notes/0-31.mdx:89]: That is fine

**Comment by keydunov** [comment]
@paveltiunov @bsod90 we need your approval here. cc @hassankhan

File: docs/content/Release-Notes/0-31.mdx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
```
Comment (paveltiunov) [inline docs/content/Release-Notes/0-31.mdx:89]: @hassankhan Sigma Computing still in the works

**Review by paveltiunov — APPROVED** [review]

**Pull request merged**